docs/migrating-from-0.9.md

```diff
@@ -197,10 +197,10 @@ export const simpleFlow = ai.defineFlow(
   },
   async (input, { context }) => {
     if (!context.auth) {
-      throw new Error('Authorization required.');
+      throw new UserFacingError('Authorization required.');
     }
     if (input.uid !== context.auth.uid) {
-      throw new Error('You may only summarize your own profile data.');
+      throw new UserFacingError('You may only summarize your own profile data.');
```

Comment:
that's not the correct use of UserFacingError:
```ts
constructor(status: StatusName, message: string, details?: any) 
```

Reply:
Oh should I change it back? We're not using UserFacingError in the sample as it

Reply:
Do:

```ts
throw new UserFacingError("UNAUTHORIZED", "Authorization required.");
throw new UserFacingError("UNAUTHORIZED", "You may only summarize your own profile data.");
```